decidim-meetings/app/cells/decidim/meetings/dates_and_map/show.erb

```diff
@@ -19,7 +19,7 @@
 
   <%= static_map %>
 
-  <% if online? %>
+  <% online? %>
```

Comment:
Uops I messed up with my suggestion and it broke all of the checks. This should be as it was before 
```suggestion
  <% if online? %>
```

Reply:
Updated in commit: https://github.com/decidim/decidim/pull/12467/commits/9500ffc71d8628577e99ebb91c8ecaa6fb97a93a